docs/semgrep-secrets/conceptual-overview.md

```diff
@@ -0,0 +1,254 @@
+---
+slug: /semgrep-secrets/conceptual-overview
+append_help_link: true
+title: Conceptual overview
```

Comment:
Hey Pablo, thanks for the review. I used "Conceptual overview" because I wasn't sure if Overview sufficiently captured what the document was about (answering "how secrets works" and our approach rather than being a procedural guide). I have this same problem in **Semgrep Supply Chain Overview** vs **Semgrep Supply Chain Getting Started** and figured "Conceptual overview" made it clearer that this is a "how-why" rather than "how-**to**".

Alternatively, we could also use "How secrets works" as a title.

Let me know what you think!

Reply:
Ah I see, that makes sense. I would keep them (Secrets and SSC) consistent then, whichever you choose. I don’t have a strong preference.

Reply:
I've updated SSC to be consistent :)